cmd/lotus-miner/storage.go

```diff
@@ -126,7 +126,7 @@ over time
 		}
 
 		if cctx.Bool("init") {
-			if err := os.MkdirAll(p, 0755); err != nil {
+			if err := os.MkdirAll(p, 0o755); err != nil {
```

Comment:
```suggestion
			if err := os.MkdirAll(p, 0755); err != nil {
```

let's not change these, `0` prefix is fine for now and we have a _lot_ of instances in the codebase of the `0` style:

```
$ git grep ', 0\(644\|755\)' | wc -l
103
```

Although there is one in storage/sealer/tarutil/systar.go with a `// nolint` that's just `644` that you could fix in this PR if you like, and remove the `// nolint`.